examples/example_config_files/ivc_inactivelayer.yaml

```
@@ -40,33 +40,39 @@ detectors:
         name: constant
         value: 5.6769e15cm^-3
     charge_trapping_model:
-      model: ConstantLifetime
+      model: Boggs
       parameters:
-        τh: 1ms
-        τe: 1ms
-        τh_inactive: 1μs
-        τe_inactive: 1μs
-      # inactive_layer_geometry:
+         nσe-1: 1020cm
+         nσh-1: 2040cm
+      #model: ConstantLifetime
+      #parameters:
+      #   τh: 1ms
+      #   τe: 1ms
+      model_inactive: ConstantLifetime
+      parameters_inactive:
+         τh_inactive: 1μs
+         τe_inactive: 1μs
```

If we include `inactive` in the `parameters` field name, I don't see why we would need it to `τe` and `τh` also, because it feels a bit redundant:
```suggestion
      model_inactive: ConstantLifetime
      parameters_inactive:
         τh: 1μs
         τe: 1μs
```